### ISSUE TITLE
Collect Usage Metrics for subscribers

### ISSUE THREAD

**Issue by johnmccawley** [Faktorz/faktorz-metering#1]
As a subscriber to the FaktorZ platform I would like to see usage metrics and charging data at a pod level, a namespace level and an account level in for CPU and Memory in 1-hour intervals.

**Comment by johnmccawley** [comment]
Usage metrics should be collected from Hawkular API on a pod level on a time interval no longer than 10 mins. The module should use the Chargebee metered billing function (https://www.chargebee.com/docs/metered_billing.html) to apply usage pricing (rating) and display the prices on the invoice.

**Comment by saurabh-b** [comment]
@johnmccawley Do you already have in mind as to where will the user see these usage metrics? I mean which specific page? and how do we go there?

**Comment by johnmccawley** [comment]
@saurabh-b Yes, currently a user can view detailed metrics via the web console on the Overview screen. Here is an example
![console-overview-06252018.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/58123edacb7ef91300e6a9dc/125516c66e82c728ace21e0d46eb9b94308d8facc2e6a946b64bf9913e0b77257b51f17b21dda164fd52690e050a1de853524d41f4f88b2fa6b53577df545aec806e5874c73847aaec8b17ad1da862da11d1e9e1d30695aa2f2ff58a87453f1b071180fa5c36a5994ccf59e406.png)

Or the user can go into a specific pod and take a look at the metrics for that pod. Here is an example:
![console-pod-metrics-06242018.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/58123edacb7ef91300e6a9dc/125516c66e82c728ace21e0d46eb9b94308d8facc2f9b047e950f5803b4f24606454f57924d7b379e70037431f1d11f6140b1b56f6eb8c23a7b4387089550cb8d93c5e25c53a11adbdd419eb08b232975dc7e7e2de139db17770a599855d3f160f198cfd543ca5984ecf59ff4de75f18.png)

The metrics available in the workspace are available for 1 week. The origin metrics pod in the namespace openshift-infra namespace automatically purges data over a week old.

For the summary metrics (used for Charging) I would like to display them here on the Cochera dashboard.

![cochera-dashboard-06242018.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/58123edacb7ef91300e6a9dc/125516c66e82c728ace21e0d46eb9b992b8791a8c2edbe50ac5fff953b426a237f56f77920d4bb6af3006b1300051bff054a5556f9f18829a2b63f798d0903bcd03c5c73c63c14fabed616b713a632c91edbe2e3d7158abd7f37ed998d4e341f001a80f65834a99b4bcd50f759ac.png)

I think it is possible that chargebee will store the data for free. We need to look into that.
